src/backend/index.js

```diff
@@ -1,37 +1,11 @@
-require('dotenv').config()
-const express = require("express");
-const http = require("http");
-const path = require("path");
-
-const API_PORT = process.env.API_PORT || 3000;
+const express = require('express');
 const app = express();
 
-// setup static content from vite build output dir in dist/ui if NODE_ENV is in development mode
-let STATIC_PATH = "../../dist/ui";
-if (process.env.NODE_ENV !== "production") {
-  STATIC_PATH = path.join(__dirname, "../../dist/ui");
-} else {
-  STATIC_PATH = path.join(__dirname, "ui");
-}
-
-app.use(express.static(STATIC_PATH));
```

Comment:
this part should be included in backend entrypoint to be able serving a static content. please add it back